python/GafferUI/AnimationUI.py

```diff
@@ -50,7 +52,9 @@
 	on other nodes.
 	""",
 
-	"nodeGadget:type", "",
+	"nodeGadget:type", "GafferUI::AuxiliaryNodeGadget",
```

Comment:
This is exactly what I was expecting to see - use "nodeGadget:type" to register the kind of UI we want. No need for AuxiliaryNode whatsoever if we're doing this anyway.